pyproject.toml

```diff
@@ -245,6 +259,9 @@ filterwarnings = [
     # should be addressed in pandas
     'ignore:np.find_common_type is deprecated.*:DeprecationWarning',
     'ignore:.*distutils Version classes are deprecated.*',
+    # TODO: add more context but linked to sentence-transformers
```

Comment:
just a reminder to address the TODO

Reply:
Right, that was Guillaume's, I'm not entirely sure what is missing here, I will need further testing